easyrsa3/easyrsa

```diff
@@ -820,6 +823,10 @@ Missing key expected at: $key_in"
 			pkcs_opts="$pkcs_opts -nokeys"
 		fi
 
+		if [ ! $want_pass ]; then
+			pkcs_opts="$pkcs_opts -password pass:"
```

Comment:
I believe this should be -nodes

Reply:
No, it shouldn't. This modification allows you to export encrypted pkcs12 with no (blank) password.